maintainers/maintainer-list.nix

```diff
@@ -3778,6 +3778,12 @@
     githubId = 307589;
     name = "Nathaniel Baxter";
   };
+  liamdiprose = {
+    email = "liam@liamdiprose.com";
+    github = "liamdiprose";
+    githubId = 1769386;
+    name = "Liam Diprose";
+  };
```

Comment:
this should be a separate commit

Reply:
Roger roger. To be honest I thought I was the slacker :stuck_out_tongue: 

Now that my country is locked down, I see no better option than to get back into open source. I'll be pushing soon.